rxlifecycle-android-lifecycle/src/main/java/com/trellow/relifecycle2/android/lifecycle/RxLifecycleObserver.java

```diff
@@ -0,0 +1,55 @@
+package com.trellow.relifecycle2.android.lifecycle;
```

Comment:
Some typos in the package name - should be `com.trello.lifecycle2.android.lifecycle`